wrappers/python/examples/calibrate_extrinsics.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+## License: Apache 2.0. See LICENSE file in root directory.
+## Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+from __future__ import print_function
+
+import pyrealsense2 as rs
+import numpy as np
+np.set_printoptions(suppress=True,precision=5)
+import cv2
+assert cv2.__version__[0] >= '3', 'The fisheye module requires opencv version >= 3.0.0'
+import os
+import json
+import argparse
+import glob
+from collections import OrderedDict
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--SN_T265', help='serial number of T265')
+parser.add_argument('--SN_D4xx', help='serial number of D4xx')
+parser.add_argument('--path', default="images", help='image path')
+parser.add_argument('--grid_H', default=9, help='grid height (inner corners)')
+parser.add_argument('--grid_W', default=6, help='grid width (inner corners)')
+parser.add_argument('--size', default=0.020, help='grid side length')
+parser.add_argument('--calibrate', default=False, help='run calibration (only)', action='store_true')
+parser.add_argument('--visualize', default=True, help='with GUI', action='store_true')
+args = parser.parse_args()
+CHECKERBOARD = (args.grid_H, args.grid_W)
```

Comment:
Hi, thanks for this, I think you would need to cast these to int. if arguments are passed, it would generate the following error
```
    ret1, corners1 = cv2.findChessboardCorners(gray1, CHECKERBOARD, None)
TypeError: an integer is required (got type str)
```
so `CHECKERBOARD = (int(args.grid_H), int(args.grid_W))` should fix it.